tlscanary/template/js/report_page.js

```diff
@@ -1,80 +1,78 @@
           + " vs Fx " + meta.base_metadata.app_version + " " + meta.base_metadata.branch;
-  var time = meta.run_start_time.split(".")[0].replace("T","-").replace(":","-").replace(":","-");
+  const time = meta.run_start_time.split(".")[0].replace("T","-").replace(":","-").replace(":","-");
   window.document.getElementById("header").innerHTML = "<h3 class=\"header\">" + desc + "<br>" + time + "</h3>";
   window.document.title = "TLS Canary Report: " + desc;
 }
 
-function makeChartTab(uriList, fieldName)
-{
+function makeChartTab(uriList, fieldName) {
   makeFieldControl(fieldName);
   resizeChartCanvas();
-  var data = getPieChartData(uriList, fieldName);
+  const data = getPieChartData(uriList, fieldName);
   drawChart(data, fieldName);
   updateChartCaption(data, fieldName);
 
-  var div = window.document.getElementById("chart_text");
+  const div = window.document.getElementById("chart_text");
   div.style.position = "absolute";
   div.style.top = "0";
   div.style.left = $("#chart_canvas").width() * 1.2 + "px";
 }
 
-function makeFieldControl(fieldName)
-{
+function makeFieldControl(fieldName) {
   var html = "";
   html += "<h3>Field:&nbsp;&nbsp;<select id=\"fieldNames\" name=\"fieldNames\" >";
   var columns = getVisibleColumns();
-  for (var i=0;i<columns.length;i++)
+  for (var i = 0;i < columns.length;i++)
   {
     html += "<option value=\"" + columns[i] + "\"";
-    if ( columns[i] == fieldName )
+    if ( columns[i] === fieldName )
     {
       html += " selected"
     }
     html += ">" + columns[i] + "</option>";
   }
   html += "</select>";
   html += "<span id=\"chart_caption\"></span>";
-  var div = window.document.getElementById("chart_text");
+  const div = window.document.getElementById("chart_text");
   div.innerHTML = html;
   window.document.getElementById("fieldNames").onchange = onFieldChange;
 }
 
-function onFieldChange(e)
-{
+function onFieldChange(e) {
   var fieldName = e.target.value;
   var uriList = getSortedRows();
   var data = getPieChartData(uriList, fieldName);
   updateChart(data, fieldName);
   updateChartCaption(data, fieldName);
 }
 
-function updateChart(uriList, fieldName)
-{
+function updateChart(uriList, fieldName) {
   window.document.myChart.destroy();
-  drawChart (uriList, fieldName);
+  drawChart(uriList, fieldName);
 }
 
-function updateChartCaption(uriList, fieldName)
-{
-  var div = window.document.getElementById("chart_text");
+function updateChartCaption(uriList, fieldName) {
+  const div = window.document.getElementById("chart_text");
   div.style.left = $("#chart_canvas").width() * 1.2 + "px";
   window.document.getElementById("chart_caption").innerHTML = "<h3>" + uriList.length + "&nbsp;unique&nbsp;value(s)</h3>";
 }
 
-function drawChart (data, fieldName)
-{
-  var c = window.document.getElementById("chart_canvas");
-  var ctx = c.getContext("2d");
-  window.document.myChart = new Chart(ctx).Pie(data, {animation:false});
+function drawChart (data, fieldName) {
```

Comment:
nit: whitespace before parenthesis

I have seen this elsewhere. Please check function definitions and calls.